shakirandagire/RecipeSearchRedux
# Add functionality for viewing ingredients

**What does this PR do?**
When the user clicks on the view ingredients link it provides the ingredients and measurements for the recipe.
**Description of Task to be completed?**
When the user clicks on the view ingredients link it provides the ingredients and measurements for the recipe.
**How to test the application**
Clone and set up the project
Use `$ git checkout ft-view-ingredients` to check out the branch containing the changes of this pull request
Use `$ npm start ` to start the application
Visit http://localhost:3000/.
